components/camera/videosource/logging/logger.go

```diff
@@ -120,11 +112,6 @@ func NewLogger() (*Logger, error) {
 
 // Start creates and initializes the logging file and periodically emits logs to it. This method is thread-safe.
 func (l *Logger) Start(ctx context.Context) error {
-	// TODO: support non-Linux platforms
-	if runtime.GOOS != linux {
-		return UnsupportedError{}
-	}
```

Comment:
If it's unsupported, the logger itself will be nil, and this function will be a noop. There is no need to say anything other "unsupported" _once_ on systems we don't yet support.